Documentation/help_files/external_data_dashboard_help.php

```diff
@@ -0,0 +1,170 @@
+<?php
+/**
+ * External Data Dashboard Help.
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author Ranganath Pathak <pathak@scrs1.org>
+ * @copyright Copyright (c) 2018 Ranganath Pathak <pathak@scrs1.org>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+ 
+use OpenEMR\Core\Header;
+
+require_once("../../interface/globals.php");
+?>
+<!DOCTYPE HTML>
+<html>
+    <head>
+    <?php Header::setupHeader();?>
+    <title><?php echo xlt("External Data Help");?></title>
+    </head>
+    <body>
+        <div class="container oe-help-container">
+            <div>
+                <center><h2><a name='entire_doc'><?php echo xlt("External Data Help");?></a></h2></center>
+            </div>
+            <div class= "row">
+                <div class="col-sm-12">
+                    <p><?php echo xlt("Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor");?>.
```

Comment:
Remove this script, or else all the nonsense constants will mess up the translations pipeline

Reply:
Done

Reply:
Still need to remove this script :)

Reply:
done ;)